src/main/java/com/minecolonies/util/AchievementUtils.java

```diff
@@ -0,0 +1,35 @@
+package com.minecolonies.util;
+
+import com.minecolonies.colony.Colony;
+import com.minecolonies.colony.permissions.Permissions;
+import net.minecraft.entity.player.EntityPlayer;
+import net.minecraft.stats.Achievement;
+
+import java.util.List;
+
+/**
+ *
+ */
+public final class AchievementUtils {
+
+    public static void syncAchievements(final Colony colony) {
+        final List<Permissions.Player> players = PermissionUtils.getPlayersWithAtleastRank(colony, Permissions.Rank.OFFICER);
+
+        final List<EntityPlayer> lPlayer = ServerUtils.getPlayersFromPermPlayer(players);
+
+        for (EntityPlayer player : lPlayer) {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Move this left curly brace to the beginning of next line of code. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3ALeftCurlyBraceStartLineCheck)
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Local variable 'player' could be declared final [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=pmd%3ALocalVariableCouldBeFinal)